jni/jni/javabridge.cpp

```diff
@@ -137,3 +137,12 @@ JNIEnv* JavaBridge::getEnvironment()
     else
         return 0;
 }
+
+std::string JavaBridge::getString( jstring aString )
+{
+    const char* s = JavaBridge::getEnvironment()->GetStringUTFChars( aString, NULL );
+    std::string theString = s;
+    JavaBridge::getEnvironment()->ReleaseStringUTFChars( aString, s );
+
+    return theString;
+}
```

Comment:
Convenience method to convert Java strings to std::string with automatic memory cleanup for the conversion process.